cabal.haskell-ci

```diff
@@ -0,0 +1,3 @@
+no-tests-no-benchmarks: False
+unconstrained:          False
+docspec:                True
```

Comment:
I did not read up yet, on what those other options do, I just wanted to include doctest.

We could also look for inspiration in [lens](https://github.com/ekmett/lens/blob/master/cabal.haskell-ci).

Reply:
I added documentation to the fields. :heavy_check_mark: